plugins/sync-font/host.js

```diff
@@ -1,10 +1,13 @@
 export function hostSyncFont(host) {
 	host.onRequest('font', () => {
-		const computedStyle = window.getComputedStyle(document.body);
```

Comment:
To me, this seems like the bug right here? Like in our LMS pages, we output this:

```css
html {
    font-size: 20px;
}
```

And that value changes when the user changes their font size preference in account settings. So could we just always grab it from the root and always set it on the root and ignore the body completely?

Reply:
Yeah, this is what started the font syncing discussion. In Portfolio we want to reflect the font size preference inside our iFrame.

We weren't sure if others depend on the current behaviour (getting computed style from `<body>` rather than `<html>`).

Reply:
I'm ok with treating this as a bug. I'm less worried after thinking about the fact that teams have to upgrade to get the new version of `ifrau` anyways and would hopefully test and see the difference. I can make the adjust to just grab the `html` font size and not introduce the new `sizeRoot` stuff if that works for everyone.